pyndl/activation.py

```diff
@@ -22,7 +22,7 @@ def activation(event_list, weights, number_of_threads=1, remove_duplicates=None,
     event_list : generator or str
         generates cues, outcomes pairs or the path to the event file
     weights : xarray.DataArray or dict[dict[float]]
-        the xarray.DataArray needs to have the dimensions 'cues' and 'outcomes'
+        the xarray.DataArray needs to have the dimensions 'outcomes' and 'cues'
```

Comment:
pyndl.ndl.ndl has also order 'cues','outcomes'. We have to keep this consistent. Why did you change the order?